google/cloud/spanner_v1/_helpers.py

```diff
@@ -170,6 +173,13 @@ def _make_value_pb(value):
             return Value(null_value="NULL_VALUE")
         else:
             return Value(string_value=value)
+    if isinstance(value, Message):
+        value = value.SerializeToString()
+        if value is None:
+            return Value(null_value="NULL_VALUE")
+        else:
+            b = base64.b64encode(value)
```

Comment:
Can we have this in a single line: `return Value(string_value=base64.b64encode(value))`